docs/en/observability/landing-page/data/related-docs-data.json

```
@@ -0,0 +1,58 @@
+[
+  {
+    "title": "Welcome to Elastic",
+    "description": "Get up and running quickly with Elastic Observabilty and more.",
+    "generic-description": "Get up and running quickly with Elastic products and solutions.",
+    "link": "{estc-welcome}/getting-started-guides.html",
+    "icon": "logoElastic"
+  },
+  {
+    "title": "Application Performance Monitoring (APM)",
+    "description": "Dive deeper into application performance monitoring.",
+    "generic-description": "Monitor software services and applications in real-time.",
+    "link": "{apm-guide-ref}/apm-overview.html",
+    "icon": "apmApp"
+  },
+  {
+    "title": "Elastic Integrations",
+    "description": "Connect your observability data to the Elastic Stack.",
+    "generic-description": "Connect your data to the Elastic Stack.",
+    "link": "https://docs.elastic.co/integrations",
+    "icon": "logoElastic"
+  },
+  {
+    "title": "Fleet and Elastic Agent Guide",
+    "description": "Use Fleet, a web-based UI, to centrally manage Elastic Agents collecting your observability data.",
+    "generic-description": "Use Fleet, a web-based UI, to centrally manage Elastic Agents.",
+    "link": "{fleet-guide}/fleet-overview.html",
+    "icon": "fleetApp"
+  },
+  {
+    "title": "Kibana Guide",
+    "description": "Build custom visualizations with your observability data.",
+    "generic-description": "Give shape to your data and navigate the Elastic Stack.",
+    "link": "{kibana-ref}/index.html",
+    "icon": "logoKibana"
+  },
+  {
+    "title": "Elasticsearch Service Documentation",
+    "description": "Manage the Elastic Stack through Elastic Cloud deployments.",
+    "generic-description": "Manage the Elastic Stack through Elastic Cloud deployments.",
+    "link": "{cloud}/ec-getting-started.html",
+    "icon": "logoCloud"
+  },
+  {
+    "title": "ECS Logging",
+    "description": "Format your logs into ECS-compatible JSON.",
+    "generic-description": "Format your logs into ECS-compatible JSON.",
+    "link": "{ecs-logging-ref}",
+    "icon": "logoElastic"
+  },
+  {
+    "title": "Elastic Common Schema (ECS) Reference",
+    "description": "Description to be written.",
+    "generic-description": "Description to be written.",
```

```suggestion
    "generic-description": "Normalize your event data to better analyze, visualize, and correlate events.",
```
Looks like this should be the same as the previous description, no?